config/hubs/utoronto.cluster.yaml

```diff
@@ -57,6 +57,15 @@ hubs:
                 name: University of Toronto
                 url: https://www.utoronto.ca/
         singleuser: &staging_jhub_singleuser
+          extraFiles:
+            github-app-private-key.pem:
+              mountPath: /etc/github/github-app-private-key.pem
+            gitconfig:
+              mountPath: /etc/gitconfig
+              stringData: |
+                [credential "https://github.com"]
+                helper = !git-credential-github-app --app-key-file /etc/github/github-app-private-key.pem --app-id 93515
```

Comment:
Was wondering where this app-id comes from, then noticed this in the utoronto old docs: `https://github.com/organizations/utoronto-2i2c/settings/apps/utoronto-jupyterhub-private-cloner`
Maybe add a comment that app 93515 is in fact the private cloner?